### ISSUE TITLE
RegExp Filter typo

### ISSUE THREAD

**Issue by spady7** [Rainbow-CPaaS/node-red-contrib-ale-rainbow#6]
Hi,
I was figuring out how to make work Reg Exp Filter with Notified_IM.
I think there is a typo in the code:

```
if ((node.filter != '') && (node.filter != undefined)) {
        var regexp = new RegExp(node.filter, 'img');
        node.log("Rainbow : Apply filter :"+node.filter);
        var res = JSON.stringify(contact).match(regexp);
        node.log("Rainbow : Filter result:"+JSON.stringify(res))
        if (res == null) return;
      }
```
I suppose after JSON.stringify should be (message) and not (contact).
If I use as it is a get some errors logs:

0|node-red | 24 Jan 10:29:03 - [info] [Notified_IM:Query HDA********] Rainbow : Apply filter :e0fb328e41740c1944051ab4467d7b0@openrainbow.com
0|node-red | ReferenceError: contact is not defined
0|node-red |     at EventEmitter.rainbow_onmessagereceived (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/ale-rainbow/ale-rainbow.js:434:34)
0|node-red |     at emitOne (events.js:116:13)
0|node-red |     at EventEmitter.emit (events.js:211:7)
0|node-red |     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/rainbow-node-sdk/lib/common/Events.js:116:31)
0|node-red |     at emitOne (events.js:116:13)
0|node-red |     at EventEmitter.emit (events.js:211:7)
0|node-red |     at EventEmitter.xmppClient.on (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/rainbow-node-sdk/lib/connection/XMPPService.js:443:51)
0|node-red |     at emitOne (events.js:116:13)
0|node-red |     at EventEmitter.emit (events.js:211:7)
0|node-red |     at Client.EventEmitter.(anonymous function) [as emit] (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/@xmpp/events/lib/EventEmitter.js:46:24)
0|node-red |     at FramedParser.listeners.element.element (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/@xmpp/connection/index.js:118:12)
0|node-red |     at emitOne (events.js:116:13)
0|node-red |     at FramedParser.emit (events.js:211:7)
0|node-red |     at FramedParser.onEndElement (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/@xmpp/plugins/websocket/lib/FramedParser.js:14:14)
0|node-red |     at SaxLtx.Parser.parser.on.name (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/@xmpp/xml/lib/Parser.js:33:14)
0|node-red |     at emitOne (events.js:116:13)
0|node-red | 24 Jan 10:29:03 - [red] Uncaught Exception:
0|node-red | 24 Jan 10:29:03 - ReferenceError: contact is not defined
0|node-red |     at EventEmitter.rainbow_onmessagereceived (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/ale-rainbow/ale-rainbow.js:434:34)
0|node-red |     at emitOne (events.js:116:13)
0|node-red |     at EventEmitter.emit (events.js:211:7)
0|node-red |     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/rainbow-node-sdk/lib/common/Events.js:116:31)
0|node-red |     at emitOne (events.js:116:13)
0|node-red |     at EventEmitter.emit (events.js:211:7)
0|node-red |     at EventEmitter.xmppClient.on (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/rainbow-node-sdk/lib/connection/XMPPService.js:443:51)
0|node-red |     at emitOne (events.js:116:13)
0|node-red |     at EventEmitter.emit (events.js:211:7)
0|node-red |     at Client.EventEmitter.(anonymous function) [as emit] (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/@xmpp/events/lib/EventEmitter.js:46:24)
0|node-red |     at FramedParser.listeners.element.element (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/@xmpp/connection/index.js:118:12)
0|node-red |     at emitOne (events.js:116:13)
0|node-red |     at FramedParser.emit (events.js:211:7)
0|node-red |     at FramedParser.onEndElement (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/@xmpp/plugins/websocket/lib/FramedParser.js:14:14)
0|node-red |     at SaxLtx.Parser.parser.on.name (/usr/local/lib/node_modules/node-red-contrib-ale-rainbow/node_modules/@xmpp/xml/lib/Parser.js:33:14)
0|node-red |     at emitOne (events.js:116:13)
0|node-red | 24 Jan 10:29:04 - [info] 
0|node-red | Welcome to Node-RED
0|node-red | ===================
0|node-red | 24 Jan 10:29:04 - [info] Node-RED version: v0.17.5
0|node-red | 24 Jan 10:29:04 - [info] Node.js  version: v8.9.1
0|node-red | 24 Jan 10:29:04 - [info] Darwin 17.3.0 x64 LE
0|node-red | 24 Jan 10:29:04 - [info] Loading palette nodes
0|node-red | 24 Jan 10:29:05 - [warn] ------------------------------------------------------
0|node-red | 24 Jan 10:29:05 - [warn] [rpi-gpio] Info : Ignoring Raspberry Pi specific node
0|node-red | 24 Jan 10:29:05 - [warn] ------------------------------------------------------

**Comment by ArfyFR** [comment]
Hi,

Yes thanks, it is already corrected in the next version I'm working on and which should be created today.

**Comment by ArfyFR** [comment]
corrected in v1.2.5
